src/Components/Task.tsx

```diff
@@ -1,15 +1,17 @@
 import React from "react";
-import "../todostyle.css";
+import "../App.css";
 
-interface Props {
+//PULL REQ INITIALIZED 
+
+export interface Props {
   id: number, 
   name: string,
   assignee: string,
   startDate: string,
   endDate: string,
-  setTodoDone: Promise<void>,
-  deleteTask: Promise<void>,
-  doneAttribute: number
+  setTodoDone: () => Promise<void>,
+  deleteTask: () => Promise<void>,
+  doneAttribute: boolean
```

Comment:
for boolean flags, we use `is`, 'has`, 'should` keywords to start the variable name depending on the context.. here we can use 'isDone`